features/support/shared_methods.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+def check_active_team(team)
```

Comment:
just thinking...
Maybe rename this to `expect_active_team(team)`, so we know we have `expectation` here?